### ISSUE TITLE
How to use local files？

### ISSUE THREAD

**Issue by lhj5426** [TagoDR/MangaOnlineViewer#382]
![image](https://github.com/TagoDR/MangaOnlineViewer/assets/30548000/6fdfe59b-54b5-447f-a2b1-ebbc3fb93b8f)
"Hello developer,
I remember that when using local files before, there was a browse button on the GitHub page to select files. Today, when I opened it, I found that there is no browse button, so I can't load local files. I have updated both scripts to the latest version and even tried refreshing the page, but I still don't see the old browse button. So, I want to ask, how can I use local files now?"

"As shown in the picture, as you can see, there is no longer the browse button on the GitHub introduction page as before."

**Comment by lhj5426** [comment]
> Yes github have moved somethings around, should be fix now, you need the main script

![image](https://github.com/TagoDR/MangaOnlineViewer/assets/30548000/ff3426b7-86d8-4474-8677-cd90c78419e1)
thank you

**Comment by lhj5426** [comment]
![image](https://github.com/TagoDR/MangaOnlineViewer/assets/30548000/dd30a19f-a017-4dcf-b2f4-295de9441b47)
"Although you fixed the browsing function, importing images or folders is not working properly."

**Comment by TagoDR** [comment]
It is working, I need more information:
what method are you using(zip, folder, images)?
what type of images(png, webp)?
what is the structure inside the inside the zip

**Comment by lhj5426** [comment]
https://github.com/TagoDR/MangaOnlineViewer/assets/30548000/5611159d-22c7-4a30-8b59-14b1ce1f53ca

**Comment by TagoDR** [comment]
I see the problem, for now it only works in Firefox, New chrome rules cant be bypassed.

**Comment by lhj5426** [comment]
> I see the problem, for now it only works in Firefox, New chrome rules cant be bypassed.

OK

> I see the problem, for now it only works in Firefox, New chrome rules cant be bypassed.

![image](https://github.com/TagoDR/MangaOnlineViewer/assets/30548000/13129775-ff45-4520-9fe5-6be1123611cf)
"Thank you, developer. It's fine to use Firefox, as long as it works. I happen to have Firefox Portable on my computer as well."

**Comment by TagoDR** [comment]
The problem is actually the rules set by GitHub, that only Chrome enforces.
I Have made a work around, with a local index.html file, see the instructions in the main page.

**Comment by lhj5426** [comment]
![image](https://github.com/TagoDR/MangaOnlineViewer/assets/30548000/ef9fd26b-90d2-4227-bc22-3c58743b3db4)
Everything is stuck here directly, and the bottom keeps spinning. Clicking the button also has no response.

**Comment by TagoDR** [comment]
Sorry, I forgot a detail, should be fine now, just update.

**Comment by lhj5426** [comment]
> Sorry, I forgot a detail, should be fine now, just update.

![image](https://github.com/TagoDR/MangaOnlineViewer/assets/30548000/a41d9631-0105-4c06-84e9-159b3f2612ab)
"Very perfect, it ran perfectly this time. |
Thank you so much for your hard work, I really appreciate it." ！！！！！

**Comment by tmby** [comment]
> Sorry, I forgot a detail, should be fine now, just update.
Still doesn't work 😭
Chrome 119.0.6045.160
![image](https://github.com/TagoDR/MangaOnlineViewer/assets/88327304/f45e9fb5-9f91-47c5-a426-3c8c460120fc)

**Comment by lhj5426** [comment]
> > Sorry, I forgot a detail, should be fine now, just update.
> 
> Still doesn't work 😭 Chrome 119.0.6045.160 ![image](https://user-images.githubusercontent.com/88327304/285444506-f45e9fb5-9f91-47c5-a426-3c8c460120fc.png)

![image](https://github.com/TagoDR/MangaOnlineViewer/assets/30548000/5d0f1bc6-6104-46fa-a660-dbf3ad8c3331)

**Comment by TagoDR** [comment]
Yes that right, I'll add a Note to remember to allow access to local files in chrome
